documentation/docs/guide/Developers/Patterns/PatternsTroubleShooting.md

```diff
@@ -0,0 +1,25 @@
+# Patterns Troubleshooting
```

Comment:
some small tweaks, but not required
- point to assets folder in the repo as this should be a constant
- information to proper escaping
- ....
However this would just be enhancements.